src/interfaces/express.ts

```diff
@@ -48,8 +48,7 @@ export function makeExpressHandler(
    * @param {Function} next - an Express continuation callback
    * @returns {ErrorMessage} - Returns the ErrorMessage instance
    */
-  function expressErrorHandler(
-      err: {}, req: express.Request, res: express.Response, next: Function) {
+  function expressErrorHandler(err: {}, req: {}, res: {}, next: Function) {
```

Comment:
Is it necessary to remove types from this function header? This function is only be exported to the user as a class field which already has a non-module-specific type.

Reply:
It is necessary.  Otherwise, there will be a compiler error in `index.ts` stating that the field `this.express` doesn't have types that match those associated with `expressErrorHandler`.